### ISSUE TITLE
Windows 10 errors registering pipeline

### ISSUE THREAD

**Issue by timroster** [ibm-garage-cloud/ibm-garage-cloud-cli#39]
When using git-bash shells on Win10 on IGC 0.5.4 the following error is seen with OpenShift clusters:

```
$ oc pipeline
? Select the type of pipeline that should be run? Tekton
Error registering pipeline: Command failed: kubectl config view -o jsonpath='{.contexts[?(@.name == "'dev-win10-054-tor/c100-e-us-east-containers-cloud-ibm-com:31243/IAM#timro@us.ibm.com'")].context.namespace}'
error: unexpected arguments: [== 'dev-win10-054-tor/c100-e-us-east-containers-cloud-ibm-com:31243/IAM#timro@us.ibm.com')].context.namespace}']
See 'kubectl config view -h' for help and examples
```

This does not occur at 0.5.2. 

It appears that https://github.com/ibm-garage-cloud/ibm-garage-cloud-cli/pull/37 introduced a regression due to missing escapes on single quotes at: 

https://github.com/ibm-garage-cloud/ibm-garage-cloud-cli/blob/a59e737e6fcbb82ebf2212816c98e6ec0454f8d2/src/services/namespace/namespace.ts#L71

**Comment by seansund** [comment]
I've addressed this in release v0.5.5

**Comment by timroster** [comment]
Retried on v0.5.5 from `npm` - seeing same error.
